### DISCUSSION TITLE
How to disable format on write

### DISCUSSION THREAD

**Discussion by aksh1618** [epwalsh/obsidian.nvim#703] [Q&A]
Hey there, thanks for this awesome plugin! I just wanted to ask if there is a way to disable the formatting that happens when writing a note (the autoformat on save, in other words). Apologies if there is an obvious way to do this, I went through issues, discussions etc. but couldn't find anything to help figure this out :see_no_evil: I'm using lazyvim, here's my configuration, just in case:

```lua
return {
    "epwalsh/obsidian.nvim",
    version = "*", -- recommended, use latest release instead of latest commit
    lazy = true,
    ft = "markdown",
    dependencies = {
        "nvim-lua/plenary.nvim",
    },
    opts = {
        disable_frontmatter = true,
        workspaces = {
            {
                name = "myvault",
                path = "~/myvault",
            },
        },
        completion = {
            nvim_cmp = true,
            min_chars = 2,
        },
    },
}
```

**Comment by malokroman** [comment]
I am having the same problem. I was trying to create dynamic tags and aliases, but the formatter cleared them all on write. Maybe I just have to write it in another editor.

**Reply by malokroman** [reply]
Okay I found an option to turn it off
```
  -- Optional, boolean or a function that takes a filename and returns a boolean.
  -- `true` indicates that you don't want obsidian.nvim to manage frontmatter.
  disable_frontmatter = false,
```

**Reply by aksh1618** [reply]
I already have it as true, last I checked, it only disables automatic insertion of frontmatter, not the formatting!

**Reply by malokroman** [reply]
> I already have it as true, last I checked, it only disables automatic insertion of frontmatter, not the formatting!

I don't think this plugin is responsible for setting autoformat. Are you using any distro for nvim?
For LazyVim, I have this line in `lua/config/options.lua`: `vim.g.autoformat = false`

I think it applies to other config as well.
